### PR TITLE
Frontend/landingpage

### PR DESCRIPTION
**Pull request by heejung-hong**

### CODE REVIEW

File: frontend/package-lock.json [hunk 1]
```diff
@@ -17230,6 +17230,19 @@
         "is-typedarray": "^1.0.0"
       }
     },
+    "node_modules/typescript": {
```
Comment (loerk) [inline frontend/package-lock.json]: ups did we add typescript? i thought we agreed in not using it.

File: frontend/src/views/AddFriend.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import styles from '../styles/Dashboard.module.css';
+import { useState } from 'react';
+
+function AddFriend() {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState();
```
Comment (loerk) [inline frontend/src/views/AddFriend.js:7]: you can give this null as first value

**Review by loerk — COMMENTED** [review]
its a bit weird that we have so many changes in the package-lock file. are you sure you are using node 18.18.2? you can check this with `node -v`

**Review by lnovitz — APPROVED** [review]

**Comment by lnovitz** [comment]
I think this is out of date now, closing

**Pull request closed without merging**